code/modules/reagents/reagent_containers/glass_containers.dm

```diff
@@ -16,6 +16,7 @@
 	resistance_flags = ACID_PROOF
 	blocks_emissive = FALSE
 	var/label_text = ""
+	usesound = 'sound/items/deconstruct.ogg'
```

Comment:
this needs to go above the `var/` declaration. Inherited vars -> Vars defined on this type